Dockerfile-linux.template

```
@@ -261,6 +261,8 @@ RUN set -eux; \
 			"libsodium-dev",
 			"libxml2-dev",
 			"openssl-dev",
+			# https://github.com/docker-library/php/pull/1552
+			if (.version | test("^8.1")) then "patch patchutils" else empty end,
```

```suggestion
			if env.version | rtrimstr("-rc") == "8.1" then "patch", "patchutils" else empty end,
```

Minor suggestion so that the two packages are added to the list individually and swap the test to be like the one on Line 329: https://github.com/docker-library/php/blob/0b6d745a64cdbeccc92571f3ae6dfebe0a43b224/Dockerfile-linux.template#L329